cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -398,7 +399,7 @@ instance ToJSON DelegationsAndRewards where
         -> HashMap Text Aeson.Value
       delegAndRwdToJson acc k (d, r) =
         HMS.insert
-          (Text.decodeLatin1 $ Ledger.serialiseRewardAcnt k)
+          (Text.decodeLatin1 $ B16.encode $ Ledger.serialiseRewardAcnt k)
```

Comment:
Perhaps I should take this opportunity to encode the stake address as Bech32 here.

Reply:
We can do that as we convert these things over to use the API types rather than the underlying ledger lib types.